apps/src/StudioApp.js

```diff
@@ -1767,14 +1775,20 @@ StudioApp.prototype.builderForm_ = function(onAttemptCallback) {
  * @param {MilestoneReport} options
  */
 StudioApp.prototype.report = function(options) {
```

Comment:
would it be possible to add a unit test or two for the new functionality / properties in report, and how they're calculated?